### PR TITLE
feat: Add TIP 4.5

### PR DESCRIPTION
**Pull request by vp-mazekine**
Add support of CantBeEvil NFT licenses

### CODE REVIEW

**Comment by rualekseev** [comment]
@vp-mazekine 

Can you change https://docs.everscale.network/standard/TIP-4 page too?

**Comment by rualekseev** [comment]
@vp-mazekine 
Do we need to make func as responsible?

File: src/standard/TIP-4/5.md [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+
+### CantBeEvil
+* Every [TIP-4.1](1.md) `NFT` contract MAY implement instance of `CantBeEvil` contract. `NFT` tokens within the same collection MAY have different license versions unless the version is specifically set in the `Collection` contract;
+* Every [TIP-4.1](1.md) `Collection` contract MAY implement instance of `CantBeEvil` contract. In such case, the derived [TIP-4.1](1.md) `NFT` contract MUST inherit the same license from the collection.
+
+#### `ICantBeEvil.sol`
+```solidity
+pragma ever-solidity >= 0.61.2;
+
+interface ICantBeEvil is TIP_6 {
```
Comment (nikitaghost) [inline src/standard/TIP-4/5.md:46]: I think we need to remove the inheritance of TIP6

**Review by ilyar — CHANGES_REQUESTED** [review]
- update idex (src/standard/TIP-4/core-description.md) status `draft`
- add responsible
- squash commit
- (optional) add link implementation

**Comment by rualekseev** [comment]
Implementation don't important for a draft, as i think.

**Comment by vp-mazekine** [comment]
* [x] update index (src/standard/TIP-4/core-description.md) status `draft`
* [x] add responsible
* [x] squash commit

File: src/standard/TIP-4/5.md (hunk 1 above)
Comment (vp-mazekine) [reply src/standard/TIP-4/5.md]: > I think we need to remove the inheritance of TIP6

Removed inheritance

**Pull request closed without merging**